contrib/sssd.spec.in

```diff
@@ -1037,5 +1037,4 @@ fi
 %systemd_postun_with_restart sssd.service
 
 %changelog
-* Thu Jan 21 2021 Pavel Březina <pbrezina@redhat.com> - @PACKAGE_NAME@-@PACKAGE_VERSION@-0@PRERELEASE_VERSION@
-- Built from upstream sources.
+%autochangelog
```

Comment:
How resulting log will look like?
IIUC, it should be built from git history, dist-git in case of Fedora/CS?
In CS we typically want to reference a list of tickets fixed...

Reply:
No